package.json

```diff
@@ -95,18 +95,20 @@
     "prettier": "1.19.1",
     "react": "16.13.0",
     "react-dom": "16.13.0",
-    "rollup": "1.32.0",
-    "rollup-plugin-babel": "4.3.3",
-    "rollup-plugin-postcss": "2.2.0",
-    "rollup-plugin-terser": "5.2.0"
+    "rollup": "2.0.2",
+    "rollup-plugin-postcss": "2.3.0",
+    "rollup-plugin-terser": "5.2.0",
+    "rollup-plugin-typescript2": "0.26.0",
+    "ts-jest": "25.2.1",
+    "tslib": "1.11.1",
+    "typescript": "3.8.3"
   },
   "peerDependencies": {
     "react": ">= 16.8.0",
     "react-dom": ">= 16.8.0"
   },
   "dependencies": {
-    "focus-options-polyfill": "1.2.0",
-    "prop-types": "15.7.2",
+    "focus-options-polyfill": "1.4.0",
```

Comment:
I appreciate you trying to upgrade this, but there's a bug in that library with regard to rendering without a DOM environment. We need to keep this at `1.2.0` until then